src/css/css-custom-property-scanner.ts

```diff
@@ -90,7 +90,7 @@ class CssCustomPropertyVisitor implements Visitor {
     if (node.type === shady.nodeType.declaration &&
         node.name.startsWith('--')) {
       this.features.push(new CssCustomPropertyAssignment(
-          node.name, this.document.sourceRangeForNode(node)));
+          node.name, this.document.sourceRangeForShadyRange(node.nameRange)));
```

Comment:
Could you add a description to `sourceRangeForShadyRange` to briefly explain why it's different to `sourceRangeForNode`?